src/integration-test/bootstrap.js

```diff
@@ -0,0 +1,3 @@
+process.env.TS_NODE_TRANSPILE_ONLY = "true"
```

Comment:
you could probably get rid of this file by using ts-node instead of node for running the bootstrap file